test/run-tests

```diff
@@ -681,6 +700,10 @@ class Task:
                         return False
 
                     print("SUCCESS")
+
+            if test_collections and not keep_results:
+                cleanup_collection_tempdirs()
```

Comment:
This is problematic because it will remove the lsr_role2collection script - you just want to clean up the collection converted role and tests here, not the lsr script too - see #145